engine/JustDecompileCmd.Shared/AssemblyInfo.cs

```diff
@@ -33,9 +33,11 @@
 /* AGPL */
 #if RELEASE
 [assembly: InternalsVisibleTo("ConsoleRunner, PublicKey=00240000048000009400000006020000002400005253413100040000010001008d3ec49d2c78b3f6029c19200a28c4a8dc279a9dfddcc7413c0737c73b49e08a3761e148e745fe2007d8d057a962c7eaf22e7eded052bb08f1e7f0d7794db4827b09124ffa61625879af57120f8078fad84c8c7d4f6c6ebb9ab14de089d606ca0ed66b9af0c67795fa4e34f61ce62732180d06fb67b3ec93b202f045e10a99d3")]
-[assembly: InternalsVisibleTo("Decompiler.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010001be52ebf131facc15d6b5893afa9799462506e031d439d70017b9c1a87147029ee619e119d71f63222c25f0910b4a2f77d6635b110a2229375e3e1e23f337eca02258c497263d9d1191af05fff461d222da5fbf2552031dca79a5341f7e00582af029c9b0d14ba113e35180fcbab106ac896c9e902dff8a76b14535218cabb0")]
+[assembly: InternalsVisibleTo("ConsoleRunnerDotNet6, PublicKey=00240000048000009400000006020000002400005253413100040000010001008d3ec49d2c78b3f6029c19200a28c4a8dc279a9dfddcc7413c0737c73b49e08a3761e148e745fe2007d8d057a962c7eaf22e7eded052bb08f1e7f0d7794db4827b09124ffa61625879af57120f8078fad84c8c7d4f6c6ebb9ab14de089d606ca0ed66b9af0c67795fa4e34f61ce62732180d06fb67b3ec93b202f045e10a99d3")]
+[assembly: InternalsVisibleTo("Decompiler.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010001be52ebf131facc15d6b5893afa9799462506e031d439d70017b9c1a87147029ee619e119d71f63222c25f0910b4a2f77d6635b110a2229375e3e1e23f337eca02258c497263d9d1191af05fff461d222da5fbf2552031dca79a5341f7e00582af029c9b0d14ba113e35180fcbab106ac896c9e902dff8a76b14535218cabb0")] // This public key is wrong

```

Comment:
If the public key is wrong that means the InternalsVisibleTo is not used at all. We can get rid of it.